source/assets/js/helpers/events.js

```diff
@@ -57,7 +57,7 @@ class WindowEventListener {
 
 		// Save to global namespace
 		$.extend(true, estatico, { events: {} });
-		estatico.events[eventName] = debouncedEventName;
+		estatico.events[debouncedEventName.split('.')[0]] = debouncedEventName;
```

Comment:
This is necessary if we want to have both `debounced` and `throttled` variants of `scroll`/`resize`. Otherwise the one registered later will overwrite the first one.

I have updated `docs/Coding_Guidelines.md` accordingly.

Is this a breaking change for anyone?